dali/base/dasess.cpp

```diff
@@ -930,6 +912,7 @@ class CClientSessionManager: public CSessionManagerBase, implements IConnectionM
     }
 
 
+    //Called by execution engine
```

Comment:
Why is this comment here?  I think all these functions are called by the engines.

Reply:
Put the comment in while developing, I was getting confused by the 2 implementations of getPermissionsLDAP.  Removed...